packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,39 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
+  const [unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);
+  const [showChat, setShowChat] = useState(false);
+  const [showPeople, setShowPeople] = useState(false);
+
   meetingAdapter.onStateChange((newState) => {
     setCurrentPage(newState.page);
     setCurrentMeetingState(newState.meeting?.state);
   });
 
-  const [showChat, setShowChat] = useState(false);
-  const [showPeople, setShowPeople] = useState(false);
+  useEffect(() => {
+    const incrementUnreadtChatMessagesCount = (event: { message: ChatMessage }): void => {
+      if (!showChat && event.message.senderDisplayName !== '') {
+        if (event.message.type === 'text' || event.message.type === 'html') {
+          setUnreadChatMessagesCount(unreadChatMessagesCount + 1);
+        }
+      }
+    };
+    meetingAdapter.on('messageReceived', incrementUnreadtChatMessagesCount);
+
+    return () => {
+      meetingAdapter.off('messageReceived', incrementUnreadtChatMessagesCount);
+    };
+  }, [meetingAdapter, setUnreadChatMessagesCount, showChat, unreadChatMessagesCount]);
 
   const closePane = useCallback(() => {
     setShowChat(false);
     setShowPeople(false);
   }, []);
 
   const toggleChat = useCallback(() => {
+    if (!showChat) {
+      setUnreadChatMessagesCount(0);
+    }
     setShowPeople(false);
     setShowChat(!showChat);
   }, [showChat]);
```

Comment:
This dependency list should ideally also include the `set*` functions being called.

Reply:
this is now in another PR: https://github.com/Azure/communication-ui-library/pull/1411